packages/components/package.json

```diff
@@ -38,9 +38,9 @@
         "moment": "^2.29.3",
         "node-fetch": "^2.6.11",
         "pdf-parse": "^1.1.1",
+        "puppeteer": "^20.7.1",
         "weaviate-ts-client": "^1.1.0",
-        "ws": "^8.9.0",
-        "html-to-text": "^9.0.5"
```

Comment:
accidentally removed `html-to-text`?

Reply:
Yeah I think I accidentally deleted it when resolving the merge conflict

Reply:
Added a fix